### PR TITLE
[border-agent] return invalid args when lifetime exceeds 10min

### PR DESCRIPTION
**Pull request by mia1yang**

### CODE REVIEW

File: src/dbus/server/dbus_thread_object_rcp.cpp [hunk 1]
```diff
@@ -70,6 +70,8 @@
 #define OTBR_CONFIG_BORDER_AGENT_MESHCOP_E_UDP_PORT 0
 #endif
 
+#define OTBR_BORDER_AGENT_MAX_EPHEMERAL_KEY_TIMEOUT (10 * 60 * 1000u)
```
[the remark this answers is not in the document]
Comment (mia1yang) [reply src/dbus/server/dbus_thread_object_rcp.cpp:73]: this definition is in the core? Can we use directly?

**Review by sunytt — APPROVED** [review]
LGTM

File: src/dbus/server/dbus_thread_object_rcp.cpp (hunk 1 above)
[the remark this answers is not in the document]
Comment (mia1yang) [reply src/dbus/server/dbus_thread_object_rcp.cpp]: Accept, thanks!

**Review by librasungirl — APPROVED** [review]

**Review by librasungirl — COMMENTED** [review]
would you please also update instropect.xml on ActivateEphemeralKeyMode dbus API for such key information?

**Review by jwhui — APPROVED** [review]

**Pull request merged**